src/layouts/Docs.tsx

```diff
@@ -150,7 +150,7 @@ const Content = (props: ChildOnlyProp) => {
     <Box
       as={MainArticle}
       flex={`1 1 ${mdBreakpoint}`}
-      maxW={{ base: "full", lg: mdBreakpoint }}
+      w={{ base: "full", lg: "0" }}
```

Comment:
@pettinarip Any thoughts on a better way to handle this? Not a fan of using a zero width, but it works exactly as I'd like so figured I'd commit it and see what others thought

Reply:
As alternatives we could set `minW="0"` or
```suggestion
      overflow="auto"
```